### DISCUSSION TITLE
Can anybody explain how to display local sensors with display batch files

### DISCUSSION THREAD

**Discussion by catdogmaus** [arendst/Tasmota#22152] [Q&A]
In https://tasmota.github.io/docs/Displays/#batch-files is talk about using batch files for display literally in 10 lines. Example only shows how display static text. There is even no real explanation about that only example. I can obviously understand that `x` and `y` are positioning but I can only guess that `f` has maybe something todo with font and `D` with display? 
I have looked at that page and did not found actually those parameters (`x,y,f,D`) anywhere.
Also about displaying local sensors only examples are with using rules.
Eventually my goal will be to show `DS18B20-1` and `DS18B20-2 `

**Comment by sfromis** [comment]
More explanation on the same page:
https://tasmota.github.io/docs/Displays/#displaytext-parameters
https://tasmota.github.io/docs/Displays/#display-json-variables
